types/mkdirp/index.d.ts

```diff
@@ -35,6 +53,46 @@ declare namespace mkdirp {
         fs?: FsImplementationSync;
     }
 
-    function sync(dir: string, opts?: Mode | OptionsSync): Made;
+    /**
+     * Synchronously create a new directory and any necessary subdirectories at
```

Comment:
We can use `code block` to highlight the code as it is done in the documentation. This is useful when reading documentation in the editor.

Here and below.